src/components/views/context_menus/MessageContextMenu.js

```diff
@@ -154,11 +154,11 @@ export default class MessageContextMenu extends React.Component {
     };
 
     onForwardClick = () => {
-        if (this.props.onCloseDialog) this.props.onCloseDialog();
-        dis.dispatch({
-            action: 'forward_event',
+        const ForwardDialog = sdk.getComponent("dialogs.ForwardDialog");
```

Comment:
As far as I know, this is an outdated way to get components. You should use an `import` at the top of the file - `import ForwardDialog from "../file/path"`

Reply:
Ah okay, I just did it for consistency with the surrounding code but can change